custom_components/google_home/translations/en.json

```diff
@@ -11,7 +11,7 @@
       }
     },
     "error": {
-      "auth": "Username/Password is incorrect. If your google account has 2FA enabled please generate app password (https://myaccount.google.com/apppasswords)"
+      "auth": "Username/Password is incorrect. If your google account has 2FA enabled please generate app password (https://myaccount.google.com/apppasswords). If authentication still fails please see https://github.com/leikoilja/ha-google-home/issues/95"
```

Comment:
I still think that we need to complete Troubleshooting section in readme instead of this link 🙂

Reply:
Good point. Let me change the link to refer to README and then we can be more elaborate in Troubleshooting section